backend/GRACEFUL_SHUTDOWN_TESTING.md

```
@@ -0,0 +1,235 @@
+### Test Long-Running Requests (>30s)
+
+If you want to test the timeout:
+
+1. Modify the routes to add a longer sleep:
+   ```go
+   time.Sleep(31 * time.Second)  // Longer than 30s timeout
+   ```
+
+2. Start a request and send SIGTERM while it's running
+3. Expected output after 30s:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)